editortools/select.py

```diff
@@ -1232,16 +1232,28 @@ def openCommands(self):
         first = True
         space = config.commands.space.get()
         sorting = config.commands.sorting.get()
+        grouping = config.commands.grouping.get()
         edit = fileEdit(filename, os.path.getmtime(filename), self.editor.selectionBox(), self.editor,
                         self.editor.level)
 
         order = []
-        if sorting == "chain":
+        if grouping.chains:
             skip = []
             done = []
             chainStored = []
             for coords in GetSort(self.editor.selectionBox(), sorting):
-                (x, y, z) = coords
+                if sorting.order == "xyz":
```

Comment:
You can now sort in any direction. However, I think I may have the orders backwards. I am not sure if XYZ sorting should be ittertools.product(x,y,z) or ittertools.product(z,y,x).

I am pretty sure I have it backwards, but I am too confused to confirm alone.